packages/ui/src/machines/authenticator/index.ts

```diff
@@ -210,6 +211,9 @@ export function createAuthenticatorMachine() {
                 s.toLowerCase()
               ) ?? [];
 
+            let cliPasswordSettings =
```

Comment:
The other examples converted the values to lower case. But I don't think that's necessary here.